Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md

```diff
@@ -175,33 +179,71 @@ To deploy a virtual machine with compute services you can edit one of windows or
 | Username | Specify a username to be able to RDP or SSH into the instance directly. |
 | Password | Specify a password to be able to RDP or SSH into the instance directly. |
 | SSH Certificate | Only in Linux machines you can specify a certificate to access via ssh. |
-| Instances | Specify the number of instances to spawn. If you increase it to a value higher than 1, a **High Availability** toggle will appear below that you can enable to use Azure availability sets for high availability support. Note that at this time, we don’t autoscale or load balance instances. To enable that, you have to manually configure these options in [Azure](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-how-to-scale-portal) |
+| Instances | Specify the number of instances to spawn. If you increase it to a value higher than 1, a **High Availability** toggle will appear below that you can enable to use Azure availability sets for high availability support. |
```

Comment:
In case of scale sets all instances belong to an automatic managed availability set